services/pkg/man-cgi/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/void-linux/infra-nginx:20220804RC02
+
+RUN xbps-install -Suy xbps && xbps-install -y void-man-cgi fcgiwrap spawn-fcgi && \
+	rm -rf /var/cache/xbps
+
+WORKDIR /man-cgi
+COPY run run
+
+COPY root.conf /etc/nginx/man-cgi.conf
+RUN sed -e '/location \//,/}/d' \
+	-e '/root \/srv\/www;/ainclude /etc/nginx/man-cgi.conf;' \
+	-i /etc/nginx/nginx.conf
```

Comment:
this feels very hacky, but purely appending an `include` directive didn't work ("location directive in the wrong place!!"). maybe a generic `include /etc/nginx/locations.d/*.conf;` directive could be included in the `infra-nginx` config?

I'll sleep on it

Reply:
basically what I'm thinking
```patch
diff --git a/services/pkg/nginx/Dockerfile b/services/pkg/nginx/Dockerfile
index bd2a651..1a6545e 100644
--- a/services/pkg/nginx/Dockerfile
+++ b/services/pkg/nginx/Dockerfile
@@ -4,4 +4,5 @@ RUN xbps-install -Suy xbps && xbps-install -Sy nginx tini
 
 WORKDIR /srv/www
 COPY nginx.conf /etc/nginx/nginx.conf
+COPY root.conf /etc/nginx/locations.d/root.conf
 ENTRYPOINT ["/usr/bin/tini", "nginx"]
diff --git a/services/pkg/nginx/nginx.conf b/services/pkg/nginx/nginx.conf
index 88a2e4f..dfcab70 100644
--- a/services/pkg/nginx/nginx.conf
+++ b/services/pkg/nginx/nginx.conf
@@ -26,8 +26,6 @@ http {
 
         root /srv/www;
 
-        location / {
-            autoindex on;
-        }
+        include /etc/nginx/locations.d/*.conf;
     }
 }
diff --git a/services/pkg/nginx/root.conf b/services/pkg/nginx/root.conf
new file mode 100644
index 0000000..27ba720
--- /dev/null
+++ b/services/pkg/nginx/root.conf
@@ -0,0 +1,3 @@
+location / {
+    autoindex on;
+}
```